DockerSettings.php

```diff
@@ -489,3 +490,6 @@
 		] );
 	};
 }
+
+# Debug mode
+$wgDebugMode = (bool)getenv('MW_DEBUG_MODE');
```

Comment:
Don't we need to enable the DebugMode extension in case when MW_DEBUG_MODE is not zero?
I mean we probably should add wfLoadExtension( 'DebugMode' ) when MW_DEBUG_MODE is not zero...